salvo/src/lib/api/env.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package salvo;
+
+// Capture all Environment variables required for the benchmark
+message EnvironmentVars {
+  // Specify the IP version for tests
+  oneof envoy_ip_test_versions {
+    bool v4only = 1;
+    bool v6only = 2;
+    bool all = 3;
+  }
+
+  // Controls whether envoy is placed between the nighthawk client and server
```

Comment:
Do we plan to have an execution mode where Envoy won't be started? Can you help me understand our plan around that?

(optional) Secondly it feels we are abusing field by giving its empty value a special meaning. It may be more explicit and readable if we have a separate explicit field, say a boolean called `include_envoy`.